docs/docs/genai/eval-monitor/ai-insights/ai-issue-discovery.mdx

```diff
@@ -0,0 +1,189 @@
+  }
+]} />
+
+The tool generates a detailed markdown report with specific trace examples, quantitative evidence, and actionable recommendations for improvement.
+
+## Usage
+
+The Analyze Experiment functionality is available through two methods:
+
+<TabsWrapper>
+<Tabs>
+<TabItem value="mcp" label="MCP" default>
+
+### Using MCP
+
+If you have MLflow's MCP server configured, you can simply run:
+
+```
+/analyze-experiment
+```
+
+### Prerequisites
+- MLflow MCP server (see [MCP setup guide](/genai/mcp/))
+- A coding agent with MCP support (e.g., Claude Code, Cursor, Windsurf, etc.) configured to connect to MLflow MCP server
+- MLflow experiment with logged traces
+
+</TabItem>
+<TabItem value="cli" label="CLI">
+
+### Using CLI
+
+Run the analyze experiment command directly:
+
+```bash
+mlflow ai-commands genai/analyze_experiment
+```
+
+**Note**: This command must be run inside a coding agent with MCP support. The agent will interactively guide you through the analysis process.
+
+### Prerequisites
+- A coding agent with MCP support (e.g., Claude Code, Cursor, Windsurf, etc.)
+- MLflow 3.4 or later: `pip install mlflow>=3.4`
+- Access to MLflow tracking server with logged traces
+
+</TabItem>
+</Tabs>
+</TabsWrapper>
+
+## Analysis Workflow
+
+<DAGLoop
+  title="Analyze Experiment Workflow"
+  circleSize={450}
+  steps={[
+    {
+      icon: GitBranch,
+      title: "Setup & Authentication",
+      description: "Configure MLflow connection",
+      detailedDescription: "Connect to your MLflow tracking server (Databricks, local, or cloud) with appropriate credentials"
+    },
+    {
+      icon: Search,
+      title: "Select Experiment",
+      description: "Choose experiment to analyze",
+      detailedDescription: "Browse and select the experiment containing traces you want to analyze for issues"
+    },
+    {
+      icon: Bot,
+      title: "Identify Agent Purpose",
+      description: "Understand agent capabilities",
+      detailedDescription: "Automatically detect what your agent does, its tools, and data sources from trace patterns"
+    },
+    {
+      icon: Lightbulb,
+      title: "Analyze Issues",
+      description: "Test hypotheses systematically",
+      detailedDescription: "Systematically analyze operational and quality issues through hypothesis-driven testing"
+    },
+    {
+      icon: FileText,
+      title: "Generate Report",
+      description: "Create actionable insights",
+      detailedDescription: "Produce comprehensive markdown report with issues, recommendations, and improvement plans"
+    }
+  ]}
+/>
+
+### 1. Setup and Authentication
+
+The tool will ask you to configure authentication:
+
+- **Databricks**: Provide workspace URL and personal access token, or use a Databricks CLI profile
+- **Local MLflow**: Specify tracking URI (SQLite, PostgreSQL, MySQL, or file store)
+- **Environment Variables**: Use existing MLflow environment configuration
```

Comment:
Nit - can we make this a bit clearer? Not 100% obvious what `existing MLflow environment configuration` is

Reply:
done! linked to the env docs